backend/api/models/FuelCode.py

```diff
@@ -0,0 +1,126 @@
+"""
+    REST API Documentation for the NRS TFRS Credit Trading Application
+
+    The Transportation Fuels Reporting System is being designed to streamline
+    compliance reporting for transportation fuel suppliers in accordance with
+    the Renewable & Low Carbon Fuel Requirements Regulation.
+
+    OpenAPI spec version: v1
+
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+from django.db import models
+from auditable.models import Auditable
+
+from api.models.FuelCodeStatus import FuelCodeStatus
+
+
+class FuelCode(Auditable):
+    """
+    Fuel codes for the Secure Document Upload
+    """
+    fuel_code = models.CharField(
+        max_length=100,
+        db_comment='Low Carbon Fuel Code.'
+    )
+    company = models.CharField(
+        max_length=100,
+        db_comment='Company.'
+    )
+    carbon_intensity = models.DecimalField(
+        blank=True,
+        null=True,
+        max_digits=999,
```

Comment:
999 seems quite large as a max length. Is there any value in decreasing the size?

Reply:
I talked to Justin about this, and it seem that 999 would be a safe number as the range can go into 3 digits.

While very rare, it's more to accommodate for it in case it does happen.

Reply:
Thanks. The way I read this was... the CI value could be up to 999 digits in length, rather than up to #999... I thought the max_digits would = 5 and decimal_places = 2. This would allow for 5 numbers in total, i.e. 999.99. Or, I could be wrong ;)

Reply:
that's a good catch. let me look it up

Reply:
thank you for catching that Kristin. Yes, it should be 5 not 999. I don't know which database I got that from, but that is a bit too high.